app/models/legal_aid_application.rb

```diff
@@ -239,7 +239,7 @@ def lowest_prospect_of_success
   end
 
   def parent_transaction_types
-    ids = transaction_types.map(&:parent_or_self).map(&:id)
+    ids = transaction_types.map { |transaction| transaction.parent_or_self.id }
```

Comment:
Code seems sound. i'd be inclined to use `tt` instead, or `transation_type` (singular) but no biggee.
```
ids = transaction_types.map { |tt| tt.parent_or_self.id }
```